packages/manager/apps/pci-kubernetes/src/components/create/Estimation.component.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import { render } from '@testing-library/react';
+import { describe, it, vi, expect, beforeEach } from 'vitest';
+import { useCatalogPrice } from '@ovh-ux/manager-react-components';
+import Estimation from './Estimation.component';
+import { wrapper } from '@/wrapperRenders';
+
+import { NodePoolPrice } from '@/api/data/kubernetes';
+
+vi.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key: string, params?: Record<string, string>) =>
+      params ? `${key} ${JSON.stringify(params)}` : key,
+  }),
+}));
+
+vi.mock('@ovh-ux/manager-react-components', () => ({
+  useCatalogPrice: vi.fn(),
+}));
+
+describe('Estimation Component', () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it('should render the estimated cost label', () => {
+    (useCatalogPrice as jest.Mock).mockReturnValue({
```

Comment:
```suggestion
    vi.mocked(useCatalogPrice).mockReturnValue({
```
jest shouldn't be used and vi.mocked does the same thing

Reply:
Yes but deleting jest.mock as a cast ask the real types for the hook and could be complicate to find